static.go

```diff
@@ -1,49 +1,64 @@
 
-  <title>Open Science Framework | Sign In</title>
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Open Science Framework | Sign In</title>
+    <link rel="icon" herf="/favicon.ico" type="image/x-icon" />
+  </head>
 
-	<link rel="stylesheet" href="//staging-accounts.osf.io/css/cas.css">
-  <link rel="icon" href="//staging-accounts.osf.io/favicon.ico" type="image/x-icon">
-
-  <!--[if lt IE 9]>
-    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.6.1/html5shiv.js" type="text/javascript"></script>
-  <![endif]--><script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-<script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js"></script><script type="text/javascript" src="https://cdnjs.cloudflare.com/ajax/libs/jquery-cookie/1.4.1/jquery.cookie.min.js"></script><style type="text/css">@media print{.lpiframeoverlay{display:none}}</style></head>
-<body id="cas"><div id="lptopspacer80974089" style="height: 40px;"></div>
-  <div id="container">
-      <header>
-
-        <a id="logo" href="" title="Open Science Framework Sign In">Open Science Framework | Sign In</a>
-
-      </header>
-      <div id="content">
-
-  <div id="msg" class="errors">
+  <body>
+    <div id="header">
+      <span><h3>Open Science Framwork | fakeCAS</h3></span>
+      </br>
+    </div>
 
-    <h2>Account has not been confirmed.</h2>
-    <p>This login email has been registered but not confirmed. Please check your email (and spam folder). <a href="https://staging.osf.io/resend/">Click here</a> to resend your confirmation email.</p>
-  </div>
+    {{if .NotRegistered}}
+    <div id="message">
+      <p>This login email has been registered but not confirmed. Please check your email (and spam folder). <a href="http://localhost:5000/resend/">Click here</a> to resend your confirmation email.</p>
+    </div>
+    {{end}}
 
-    <div class="row" style="text-align: center;">
-        <hr>
-        <a href="https://staging.osf.io/">Back to OSF</a>
+    {{if .NotAuthorized}}
+    <div id="message">
+      <p>The service you attempted to authenticate to is not authorized to use CAS.</p>
     </div>
-</div> <!-- END #content -->
+    {{end}}
 
-<footer>
-    <div class="copyright">
-        <div class="row">
-            <p>Copyright © 2011-2015 <a href="http://centerforopenscience.org">Center for Open Science</a> |
-                <a href="https://github.com/CenterForOpenScience/centerforopenscience.org/blob/master/TERMS_OF_USE.md">Terms of Use</a> |
-                <a href="https://github.com/CenterForOpenScience/centerforopenscience.org/blob/master/PRIVACY_POLICY.md">Privacy Policy</a>
-            </p>
-        </div>
+    {{if .NotValid}}
+      <p>Invalid request. User does not exist or invalid/expired validation key.</p>
+    {{end}}
+    {{if .LoginForm}}
+    <div id="forms">
+      <form id="login" action="/login?service=http://localhost:5000" method="post">
```

Comment:
do I need to encode if I use `url.Parse("http://" + *Host + "/login?service=" + service)`?